bot/services/ChatBot.cs

```diff
@@ -214,6 +214,11 @@ private async Task CompleteGenerationAsync(HttpClient httpClient, string userId,
         switch (chatResponse.Intent)
         {

```

Comment:
Rather than trying to interpret the status codes which could still mean that the LLM is generating something (think what if there were multiple indexes and this was just no documents from one of them), I would probably try and look at the end of the process and see if the summaries is empty.

Reply:
I did think about doing it outside of the streaming code, however, it seemed this block is responsible for responding back to the user, so it would make sense to keep the response here versus outside. I think the approach of interpreting status code should be fine since that's what they are for.

Reply:
The status messages aren't guaranteed to be anything specific though - they are just messages back to the user and in some cases are skipped if multiple messages are coming at the same time.

Other messages are send to the user and are outside the streaming block.

Reply:
updated per discussion